README.md

```diff
@@ -1042,15 +1042,13 @@ Such request can quickly generate a lot of queries on the storage handler. To en
 
 ## Pagination
 
-Pagination is supported on collection URLs using `page` and `limit` query-string parameters. If you don't define a default pagination limit using `PaginationDefaultLimit` resource configuration parameter, the resource won't be paginated until you provide the `limit` query-string parameter.
+Pagination is supported on collection URLs using the `page` and `limit` query-string parameters, and can be used for resource list view URLs with request method `GET` and `DELETE`. If you don't define a default pagination limit using `PaginationDefaultLimit` resource configuration parameter, the resource won't be paginated for list `GET` requests until you provide the `limit` query-string parameter. The `PaginationDefaultLimit` does not apply to list `DELETE` requests, but the `limit` and `page` parameters may still be used to delete a subset of items.
 
 If your collections are large enough, failing to define a reasonable `PaginationDefaultLimit` parameter may quickly render your API unusable.
 
 ## Skipping
 
-Skipping of resource items is defined through the `skip` query-string parameter. The `skip` value is a positive integer defining the number of items to skip when querying for items.
-
-To use a resource field with the `skip` parameter, the field must be defined on the resource.
```

Comment:
> To use a resource field with the `skip` parameter, the field must be defined on the resource.

@rs, I was confused by the meaning of this, and I have removed it for now. What is it trying ti say? What is resource field in this context? A schema field? If we want to add it back, a corresponding example may be appropriate (at least for me), so that it can e better understood.

Reply:
I think it is no longer relevant.